### ISSUE TITLE
Attributes that are dates read out as a single number

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#560]
Example 20221211 reads as 20 million two hundred and twenty two thousand, two hundred eleven.

This should be printed in a proper date format and be read back in a proper date format.

**Comment by bryce-mcmath** [comment]
This has been fixed in most locations, however in the proof request screen (what the holder sees) formatting info is not being provided - and it's quite complicated to determine why not exactly. We're going to open a new ticket to refactor that logic and get it working correctly so that's in a central location and at that time we'll finish formatting the dates.

**Comment by wadeking98** [comment]
I think this is resolved in the case where the credential being requested has an oca bundle associated with it. The OCA formatting will tell the wallet that it's a date and the wallet applies the proper formatting

**Comment by jleach** [comment]
Moving to QA will ask @knguyenBC to confirm if its fixed or not.

**Comment by knguyenBC** [comment]
If there's OCA formatting then the date is read outload properly. Tested both android and iOS. Dates are NOT read aloud properly when there isn't OCA formatting, for example, if a proof request is asking for credentials the holder does not have. Should the latter requirement be a separate ticket?

**Comment by jleach** [comment]
I think it is fine as-is because all our VCs will have OCA. But, final decision is @cvarjao to make.

**Comment by knguyenBC** [comment]
The problem is when the OCA isn't displayed, such as when a proof request is asking something a person doesn't have.

**Comment by knguyenBC** [comment]
relates to #1856. This ticket might be resolved if we pull the OCA for missing credentials. I'll close this ticket
